vaadin-text-field-flow-parent/vaadin-text-field-flow/src/test/java/com/vaadin/flow/component/textfield/tests/NumberFieldTest.java

```diff
@@ -20,6 +20,8 @@
 
 import java.util.Arrays;
 
+import com.vaadin.flow.component.textfield.TextFieldVariant;
```

Comment:
nit: please sort imports in this and other test files to be placed together with other `com.vaadin` imports.

Reply:
Aligned import order between test files.